tests/Earthfile

```
@@ -230,6 +231,10 @@ copy-test:
         echo "sub" > in/sub/file
     DO +RUN_EARTHLY --earthfile=copy.earth --output_does_not_contain="which does not expand to a home directory"
 
+copy-test-empty-src:
+    DO +RUN_EARTHLY --earthfile=copy.earth --target=+copy-empty-src --should_fail=true --output_contains="empty COPY source encountered"
+    DO +RUN_EARTHLY --earthfile=copy.earth --target=+copy-empty-src-if-exists --should_fail=true --output_contains="empty COPY source encountered"
```

@idodod got me thinking, I think this should pass when the `--if-exists` flag is set.
```suggestion
    DO +RUN_EARTHLY --earthfile=copy.earth --target=+copy-empty-src-if-exists
```